django_toolkit/fallbacks/circuit_breaker/circuit_breaker.py

```diff
@@ -114,3 +122,6 @@ def _increase_request_count(self):
             )
 
         self.cache.incr(self.rule.request_cache_key)
+
+
+circuit_breaker = CircuitBreaker
```

Comment:
I'm not sure about this line. It introduces two names to import the `CircuitBreaker` class. Without this line the decorator is still achieavable with:
```python
from django_toolkit.fallbacks.circuit_breaker import CircuitBreaker

@CircuitBreaker
def some_func():
    pass
```

If someone thinks its weird to use a decorator with a name which looks like a class, it can be addressed with:
```python
from django_toolkit.fallbacks.circuit_breaker import CircuitBreaker as circuit_breaker

@circuit_breaker
def some_func():
    pass
```

What do you think?

Reply:
For me, it's just syntax sugar, What do you think @jairhenrique and @jarussi?

Reply:
Either way is acceptable for me, I just would like to take a conscious decision here.

Reply:
Ok, I'll merge